hackathon_site/hardware/test_api.py

```diff
@@ -194,3 +194,244 @@ def test_hardware_get_success(self):
         data = response.json()
 
         self.assertEqual(expected_response, data["results"])
+
+
+class HardwareListTestCase(SetupUserMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        # Create 3ish hardware objects
```

Comment:
Can remove this comment, that was just me suggesting to you that you might need to create more than 3 objects (doesn't seem like you need to)